components/OrderedList/src/index.scss

```diff
@@ -4,6 +4,7 @@
   @extend .denhaag-unordered-list;
 }
 
-.denhaag-ordered-list .denhaag-unordered-list {
+.denhaag-ordered-list .denhaag-unordered-list,
+.denhaag-ordered-list ul {
```

Comment:
If all fallback must be removed, this `ul` must be removed aswel

Reply:
@paulAcato done, removed it